operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -175,6 +187,23 @@ The Technical Product Writer/s form part of the PR review/approval process. When
 
 The [Documentation Style guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide) and [UI text guidelines](https://docs.mattermost.com/process/documentation-UItext-guidelines.html).
 
+## Customer Education Processes
+
+### Resolve gaps and painpoints among customers
+
+To resolve gaps and pain points, identify and rank them in priority order.
+
+#### Identify
+
+- Monitor support tickets and get regular feedback from Support Engineers through the Support channel.
```

Comment:
Is there specific feedback that's obtained, such as a daily summary? Or just general trend-spotting?

Reply:
not yet, but having a specific predefined feedback system in place would be an ideal goal.

Reply:
Sounds good to me. :)